backend/DevUp/src/DevUp.Domain/Organization/Services/TeamService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using DevUp.Domain.Organization.Entities;
+using DevUp.Domain.Organization.Repositories;
+using DevUp.Domain.Organization.Services.Exceptions;
+using DevUp.Domain.Organization.ValueObjects;
+
+namespace DevUp.Domain.Organization.Services
+{
+    internal class TeamService : ITeamService
```

Comment:
We inject ITeamService for MediatR for all queries/commands and delegate validations here, but not for `UpdateTeamCommandHandler` which uses ITeamRepository directly, even tho it has validations. Why is that?